packages/botonic-nlu/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@botonic/nlu",
+  "version": "0.1.0",
+  "main": "lib/index",
+  "scripts": {
+    "build": "rm -rf lib && babel src -d lib",
+    "test": "jest"
+  },
+  "jest": {
+    "testEnvironment": "node"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.5.5",
+    "@tensorflow/tfjs": "^1.2.7",
+    "@tensorflow/tfjs-node": "^1.2.7",
+    "@types/jest": "^24.0.17",
+    "colors": "^1.3.3",
+    "compromise": "^11.13.2",
+    "compromise-plugin": "0.0.9",
+    "franc": "^4.1.0",
+    "fs": "0.0.1-security",
+    "inquirer": "^6.3.1",
+    "jest": "^24.8.0",
+    "sqlite-async": "^1.0.11"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-transform-runtime": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "core-js": "^3.1.4"
```

Comment:
can you try without core-js?

Reply:
in fact we have not used core-js at all in our .babelrc, so yes, removed!